trailblazer/store/models.py

```diff
@@ -103,7 +103,7 @@ def to_dict(self) -> dict:
         """Return a dictionary representation of the object."""
         return {
             "id": self.id,
-            "family": self.family,
+            "case_id": self.case_id,
```

Comment:
This will likely require changes in at least the cigrid frontend and potentially in cg as well

Reply:
https://github.com/Clinical-Genomics/cg/pull/2777. Could you take a look at the FE?